cli/command/formatter/service.go

```diff
@@ -19,9 +19,9 @@ Labels:
 {{- range $k, $v := .Labels }}
  {{ $k }}{{if $v }}={{ $v }}{{ end }}
 {{- end }}{{ end }}
-Mode:
-{{- if .IsModeGlobal }}		Global
-{{- else }}		Replicated
+Service Mode:
+{{- if .IsModeGlobal }}	Global
+{{- else }}	Replicated
```

Comment:
I think the extra tab was so that `Global` or `Replicated` will line up with the other values in the column. For example:

```
Mode:           Replicated
Replicas:       4
```